core/src/create.jl

```diff
@@ -225,13 +210,47 @@ end
 
 function FlowBoundary(db::DB, config::Config)::FlowBoundary
     static = load_structvector(db, config, FlowBoundaryStaticV1)
-    defaults = (; active = true)
-    static_parsed = parse_static(static, db, "FlowBoundary", defaults)
-    return FlowBoundary(
-        static_parsed.node_id,
-        static_parsed.active,
-        static_parsed.flow_rate,
-    )
+    time = load_structvector(db, config, FlowBoundaryTimeV1)
+
+    static = load_structvector(db, config, FlowBoundaryStaticV1)
+    time = load_structvector(db, config, FlowBoundaryTimeV1)
+
+    static_node_ids = Set(static.node_id)
+    time_node_ids = Set(time.node_id)
+    msg = "Flowboundary cannot be in both static and time tables"
+    @assert isdisjoint(static_node_ids, time_node_ids) msg
+    static_node_ids = collect(static_node_ids)
+    time_node_ids = collect(time_node_ids)
```

Comment:
Why do you need to collect here?

Reply:
Because I need to convert from `Set` to `Vector` for other operations later to work, I think `findfirst(static_node_ids .== node_id)`. It's a bit silly though, I'll use `static.node_id` and `time.node_id` directly.